From richard@backtrace.co.uk  Mon Apr 26 14:10:17 2010
From: Richard Airlie <richard@backtrace.co.uk>
Date: Mon, 05 Apr 2010 22:22:46 +0100
Subject: staging: rtl8192su: add Support for Belkin F5D8053 v6
To: greg@kroah.com
Message-ID: <4BBA54A6.4000706@backtrace.co.uk>


Please find attached a patch which adds the device ID for the Belkin
F5D8053 v6 to the rtl8192su driver. I've tested this in 2.6.34-rc3
(Ubuntu 9.10 amd64) and the network adapter is working flawlessly.

From: Richard Airlie <richard@backtrace.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/rtl8192su/r8192U_core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rtl8192su/r8192U_core.c
+++ b/drivers/staging/rtl8192su/r8192U_core.c
@@ -120,6 +120,7 @@ static const struct usb_device_id rtl819
 	{USB_DEVICE(0x07aa, 0x0043)},
 	/* Belkin */
 	{USB_DEVICE(0x050d, 0x805E)},
+	{USB_DEVICE(0x050d, 0x815F)}, /* Belkin F5D8053 v6 */
 	/* Sitecom */
 	{USB_DEVICE(0x0df6, 0x0031)},
 	/* EnGenius */
